### PR TITLE
[WIP] TMI-rebased test fixes - Intermediate CI run for tests

### PR DESCRIPTION
**Pull request by derekpierre**
Related to #2173 
- [x] Ensure created ursula is stopped; fixes tests/acceptance/cli/ursula/test_local_keystore_integration.py failure.
- [x] Ensure dry-run Ursula is stopped; fixes tests/acceptance/cli/ursula/test_stakeholder_and_ursula.py failure.

### CODE REVIEW

File: tests/acceptance/cli/ursula/test_local_keystore_integration.py [hunk 1]
```diff
@@ -147,15 +147,18 @@ def test_ursula_and_local_keystore_signer_integration(click_runner,
...
+
+        # Show that we can produce the exact same signer as pre-config...
+        assert pre_config_signer.path == ursula.signer.path
+
+        # ...and that transactions are signed by the keytore signer
+        receipt = ursula.commit_to_next_period()
+        transaction_data = testerchain.client.w3.eth.getTransaction(receipt['transactionHash'])
+        assert transaction_data['from'] == worker_account.address
+    finally:
+        ursula.stop()
```
Comment (KPrasch) [inline tests/acceptance/cli/ursula/test_local_keystore_integration.py:164]: :+1: ( although still somewhat confused about disenchant vs. stop @jMyles )

File: nucypher/cli/commands/ursula.py [hunk 2]
```diff
@@ -392,10 +392,15 @@ def run(general_config, character_options, config_file, interactive, dry_run, me
...
-                      interactive=interactive,
-                      prometheus_config=prometheus_config)
+    # TODO should we just not call run at all for "dry_run"
+    try:
+        URSULA.run(emitter=emitter,
+                   start_reactor=not dry_run,
+                   interactive=interactive,
+                   prometheus_config=prometheus_config)
+    finally:
+        if dry_run:
```
Comment (KPrasch) [inline nucypher/cli/commands/ursula.py:402]: Why only in the case of a dry run?

**Comment by KPrasch** [comment]
Included in #2173

File: nucypher/cli/commands/ursula.py (hunk 2 above)
Comment (derekpierre) [reply nucypher/cli/commands/ursula.py:402]: If Ursula is starting from the command line, then we don't want to stop it other than for dry-run.

Comment (KPrasch) [reply nucypher/cli/commands/ursula.py:402]: Perhaps I am missing something - but why not?  At the end of this test it seems like we can stop the ursula no mater what the situation is.

Comment (derekpierre) [reply nucypher/cli/commands/ursula.py:402]: If I'm understanding you correctly - you are correct, but the code change is not to the test, but rather the ursula command module itself.

**Pull request closed without merging**